reflex/app.py

```diff
@@ -498,11 +520,18 @@ def compile(self):
                         ),
                     )
                 )
+                # add component.get_imports() to all_imports
+                all_imports.update(component.get_imports())
+
                 # Add the custom components from the page to the set.
                 custom_components |= component.get_custom_components()
+
         thread_pool.close()
         thread_pool.join()
 
+        # install frontend packages
+        self.get_frontend_packages(all_imports)
```

Comment:
is there a reason to not just do this once at the end, after the custom component imports have been catalogued at well?